Docs: Add usage insights export feature (#30376)
grafana/grafana: docs/sources/enterprise/usage-insights/dashboard-datasource-insights.md

+++
title = "Dashboard and data source insights"
description = "Understand how your dashboards and data sources are used"
keywords = ["grafana", "usage-insights", "enterprise"]
aliases = ["/docs/grafana/latest/enterprise/usage-insights/dashboard-datasource-insights.md"]
weight = 200
+++

# Dashboard and data source insights

For every dashboard and data source, you can access usage information.

## Dashboard insights

> **Note:** Available in Grafana Enterprise v7.0+.

> To see dashboard usage information, from the top bar, click **Dashboard insights**.

Reviewer: ```suggestion
To see dashboard usage information, go to the top bar and click **Dashboard insights**.
```

{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_button.png" max-width="400px" class="docs-image--no-shadow" >}}

> Dashboard insights displays the following information:

Reviewer: ```suggestion
Dashboard insights show the following information:
```

> - **Stats:** Shows the number of daily queries and errors for the past 30 days.

Reviewer: ```suggestion
- **Stats:** The number of daily queries and errors for the past 30 days.
```

> - **Users & activity:** Shows the daily view count for the last 30 days; last activities on the dashboard and recent users (with a limit of 20).

Reviewer: ```suggestion
- **Users & activity:** The daily view count for the last 30 days; last activities on the dashboard and recent users (with a limit of 20).
```

{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_stats.png" max-width="400px" class="docs-image--no-shadow" >}}{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_users.png" max-width="400px" class="docs-image--no-shadow" >}}

## Data source insights

> **Note:** Available in Grafana Enterprise v7.3+.

> Data source insights give you information about how a data source has been used in the past 30 days:

Reviewer: ```suggestion
Data source insights give you information about how a data source has been used in the past 30 days, such as:
```

- Queries per day
- Errors per day

> - Average load duration per day (ms)

Reviewer 2: What does "load duration" mean?

Reviewer 3: the amount of time each query takes to load, averaged over that day.

Reviewer 2: Does it make more or less sense to use, "Query load time per day (averaged in ms)"?

> Find data source insights:

Reviewer: ```suggestion
To find data source insights:
```

+++
title = "Dashboard and data source insights"
description = "Understand how your dashboards and data sources are used"
keywords = ["grafana", "usage-insights", "enterprise"]
aliases = ["/docs/grafana/latest/enterprise/usage-insights/dashboard-datasource-insights.md"]
weight = 200
+++

# Dashboard and data source insights

For every dashboard and data source, you can access usage information in one click.

## Dashboard insights

> **Note:** Available in Grafana Enterprise v7.0+.

You can see dashboard usage information by clicking on the `Dashboard insights` button in the top bar.

{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_button.png" max-width="400px" class="docs-image--no-shadow" >}}

> It shows two kinds of information:

Reviewer 2: This phrase change eliminates a potential text bug where two becomes three or something else and the number is embedded into the phrase. Here is a great example of where we technical writers make the general comment, "Assume your readers can count."

> - **Stats:** Shows the daily query count and error count for the last 30 days.

Reviewer 2: ```suggestion
- **Stats:** Count the number of daily queries, and errors, for the past 30 days.
```

Reviewer 2: Is this daily errors or one month’s worth of errors?

Reviewer 2: @oddlittlebird Wondering about our style here, because this is not actually where the counting is happening (I think). What is your take on imperative versus not?

Reviewer 2: Same goes for line that follows.

Author: It's a daily number of errors, you can see what is displayed to the users here: https://grafana.com/static/img/docs/enterprise/dashboard_insights_stats.png
I made an update but not exactly the one you suggested.

- **Users & activity:** Shows the daily view count for the last 30 days; last activities on the dashboard and recent users (with a limit of 20).

{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_stats.png" max-width="400px" class="docs-image--no-shadow" >}}{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_users.png" max-width="400px" class="docs-image--no-shadow" >}}

## Data source insights

> **Note:** Available in Grafana Enterprise v7.3+.

> Data source insights give you information about how a data source has been used in the last thirty days.

Reviewer 2: ```suggestion
Data source insights give you information about how a data source has been used (daily) in the past 30 days:
```

> - Queries per day

Reviewer 2: ```suggestion
- Queries
```

Author: What about `Query count` or `Number of queries`? Writing only `Queries` let me think that I could access all the queries that were made in the past 30 days.
You can see the actual display here: https://grafana.com/static/img/docs/enterprise/datasource_insights.png

Reviewer 2: I changed my mind after seeing the image. Keep as is.

> - Errors per day

Reviewer 2: ```suggestion
- Errors
```

> - Average load duration per day (ms)

Reviewer 2: ```suggestion
- Average load duration in milliseconds
```

> To find data source insights, go to:

Reviewer 2: ```suggestion
To find data source insights, go to:
```
```suggestion
Find data source insights:
```

> 1. Data source list view.

Reviewer 2: ```suggestion
1. Go to the Data source list view.
```

1. Click on a data source.

> 1. Click the Insights tab.

Reviewer 2: ```suggestion
1. Click the **Insights** tab.
```
